scripts/external_deps/requirements.txt

```diff
@@ -0,0 +1,3 @@
+voluptuous==0.13.1
+jinja2==3.1.2
+pyyaml==6.0
```

Comment:
`protobuf` package is also required for building grpc library. Although it is unrelated to `external-deps` generation. Maybe add it to `grpc/CMakeLists.txt`, `grpc/requirements.txt`? Right before `include(GrpcTargets)` should be OK